templates/exchange.template.yaml

```diff
@@ -4219,6 +4240,25 @@ Resources:
            ConditionKey: "Backup"
            ConditionValue: "yes"
       BackupPlanId: !Ref ExchangeBackupPlan
+  ExchangeResourceGroup:
+    Condition: AppInsightsMonitoring
```

Comment:
There is a typo here. The condition has been defined as `appInsightsMonitoring`. Also, could you rename the condition to AppInsightsEnabled to match the main template.